### DISCUSSION TITLE
Can we hosting Decap CMS outside Netlify?

### DISCUSSION THREAD

**Discussion by randidev** [decaporg/decap-cms#6737] [Q&A] [answered]
Hi guys, I have a website that is currently integrated with NetlifyCMS / DecapCMS, but we do have some issues with Chinese users when we host it in Netlify so I was planning to host it outside Netlify, do I able to do that?

**Comment by gvrrxtt** [comment, marked as answer]
Decap CMS, just like Netlify CMS, should work just about anywhere. You may have to configure the backend config settings a little differently. Check out these pages from the docs:

https://decapcms.org/docs/add-to-your-site/

https://decapcms.org/docs/backends-overview

**Reply by randidev** [reply]
@gvrrxtt quick question, if we self-hosted the CMS, can it be like in Netlify automatically making a commit to GitHub whenever the content is edited? or do we need to create it by ourself?

**Reply by timwis** [reply]
@randidev Yes, absolutely. Among the 'backends' that @gvrrxtt listed are at least two that make commits to GitHub.com (Git Gateway and GitHub). Here are some [installation instructions](https://github.com/timwis/jkan/blob/main/docs/installation.md) for a static site that uses the GitHub backend, if helpful.

**Reply by randidev** [reply]
@timwis so I need to install and configure Jekyll too to be able to self-hosted my DecapCMS, right?
